frontend/app/components.d.ts

```diff
@@ -1,8 +1,9 @@
 // generated by unplugin-vue-components
 // We suggest you to commit this file into source control
 // Read more: https://github.com/vuejs/vue-next/pull/3399
+import '@vue/runtime-core'
 
-declare module 'vue' {
+declare module '@vue/runtime-core' {
```

Comment:
I don't remember this file before. Did you intend to add it?

Reply:
I removed it 👍